### DISCUSSION TITLE
react-table.js.org documentation

### DISCUSSION THREAD

**Discussion by NSegal** [TanStack/table#1990] [General]
I'm searching the internet for the old react-table documentation but it looks like the website [https://react-table.js.org](react-table.js.org) just redirects to it's Github page now.

We use a 2019 version of react-table and might not be able to update to the latest since we already built everything using the older version. 

[Archive.org](https://web.archive.org/web/20180918131116/https://react-table.js.org/#/story/select-table-hoc) isn't rendering much from react-table.js.org. 

Mainly I was looking for a way to return the number of table rows after [defaultFilter](https://web.archive.org/web/20180918131113/https://github.com/react-tools/react-table) and [filterMethod](https://web.archive.org/web/20180918131113/https://github.com/react-tools/react-table) have filtered the table data. 

Is there another place I can access the old documentation or do you know of a way to return the filtered row count?   

Thank you in advance!

**Comment by zecamacho** [comment]
This is the documentation for version 6 of react-table. Is this what you're looking for?

https://www.npmjs.com/package/react-table-6

**Comment by NSegal** [comment]
I'm more looking for the [react-table.js.org](https://web.archive.org/web/20180918131116/https://react-table.js.org/#/story/select-table-hoc) documentation. Appreciate the response though, Zecamacho.
